src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/CSharpVirtualDocument.cs

```diff
@@ -22,20 +18,4 @@ internal class CSharpVirtualDocument(ProjectKey projectKey, Uri uri, ITextBuffer
     internal ProjectKey ProjectKey => _projectKey;
 
     protected override CSharpVirtualDocumentSnapshot GetUpdatedSnapshot(object? state) => new(_projectKey, Uri, TextBuffer.CurrentSnapshot, HostDocumentVersion);
-
-    public override VirtualDocumentSnapshot Update(IReadOnlyList<ITextChange> changes, int hostDocumentVersion, object? state)
-    {
-        var result = base.Update(changes, hostDocumentVersion, state);
-
-#if DEBUG
-        var text = TextBuffer.CurrentSnapshot.GetText();
-
-        var generatedFileStartIndex = text.IndexOf("#pragma warning disable 1591");
-        var secondGeneratedFileStartIndex = text.IndexOf("#pragma warning disable 1591", generatedFileStartIndex + 20);
-
-        Debug.Assert(secondGeneratedFileStartIndex == -1, "Generated C# file appears to have duplicated file contents. This could indicate a sync problem between language server and client.");
-#endif
```

Comment:
Hmmm... what a strange calculation. I'm not sad to see it go. :smile:

Reply:
I thought it was kinda clever 😛